src/ext/ddtrace.c

```diff
@@ -120,7 +120,7 @@ static PHP_MINIT_FUNCTION(ddtrace) {
 
     register_span_data_ce(TSRMLS_C);
 
-    ddtrace_dispatch_inject(TSRMLS_C);
+    ddtrace_opcode_minit();
     ddtrace_compile_minit();
```

Comment:
What do you think about exposing just `ddtrace_engine_hooks_minit` and `_mshutdown` instead of having `_minit` for opcodes, compile, etc individually?

Reply:
I love that idea. Done! :)